### PR TITLE
NS: clean geometric factors

### PR DESCRIPTION
**Pull request by valeriabarra**
As per our conversation, it is cleaner to adopt a notation `x_i,j` (Jacobian `dx_i / dX_j` in indicial notation) for the change of coordinate matrix, and consequently, `X_i,j` for its inverse.

### CODE REVIEW

File: examples/navier-stokes/densitycurrent.h [hunk 1]
```diff
@@ -222,56 +222,53 @@ static int DC(void *ctx, CeedInt Q,
...
+                                   {qdata[7][i],
+                                    qdata[8][i],
+                                    qdata[9][i]}
                                   };
+    // -- Grad-to-Grad qdata
+    // ---- dXdx_j,k * dXdx_k,j
+    CeedScalar dXdxdXdxT[3][3];
+    for (int j=0; j<3; j++) {
+      for (int k=0; k<3; k++) {
+        dXdxdXdxT[j][k] = dXdx[j][k] * dXdx[k][j];
```
Comment (jedbrown) [inline examples/navier-stokes/densitycurrent.h]: It's supposed to be matrix multiplication, so perhaps `dXdx[j][l] * dXdx[k][l]` (summed over `l`).

Comment (valeriabarra) [reply examples/navier-stokes/densitycurrent.h]: Of course! Sorry about that!

**Comment by valeriabarra** [comment]
> I added a commit with minor style consistency updates (though I don't feel super strongly about them) and a documentation update for `w detJ` vs only `detJ`. This looks good to me though.

I had aligned all the q's and dq's to start at the same column with a multi-row cursor

**Comment by jeremylt** [comment]
I can add that back. I had them aligned with themselves  but not each other after realigning the equal signs.

File: examples/navier-stokes/densitycurrent.h [hunk 2]
```diff
@@ -243,30 +243,34 @@ static int DC(void *ctx, CeedInt Q,
...
     // ---- dXdx_j,k * dXdx_k,j
     CeedScalar dXdxdXdxT[3][3];
-    for (int j=0; j<3; j++)
-      for (int k=0; k<3; k++)
-        dXdxdXdxT[j][k] = dXdx[j][k] * dXdx[k][j];
+      for (int j=0; j<3; j++) {
+        for (int k=0; k<3; k++) {
+          dXdxdXdxT[j][k] = 0;
+          for (int l=0; l<3; l++)
+            dXdxdXdxT[j][k] += dXdx[j][l]*dXdx[k][l];
```
Comment (valeriabarra) [inline examples/navier-stokes/densitycurrent.h]: @jedbrown I did the vanilla implementation without loop-tiling. Do we want to do that? I wasn't sure

**Comment by valeriabarra** [comment]
> I can add that back. I had them aligned with themselves but not each other after realigning the equal signs.

I realigned q, dq's and qdata

File: examples/navier-stokes/densitycurrent.h (hunk 2 above)
Comment (jedbrown) [reply examples/navier-stokes/densitycurrent.h]: We're expecting the compiler to unroll and transform all of this anyway. (Also, tiling is a cache optimization, but this is so small it fits in registers.)

**Pull request merged**